.github/actions/backup-restore-test/action.yml

```diff
@@ -56,7 +56,7 @@ runs:
             tee $GITHUB_OUTPUT
 
     - name: Restore ${{ inputs.service }} ${{ inputs.kind }}
-      if: ${{ inputs.restore-args != "" }}
+      if: ${{ inputs.restore-args != 0 }}
```

Comment:
I think maybe we just want `inputs.restore-args`? Or maybe drop the `${{ }}`? From the GitHub actions [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif)

>When you use expressions in an if conditional, you may omit the ${{ }} expression syntax because GitHub Actions automatically evaluates the if conditional as an expression. Using the ${{ }} expression syntax turns the contents into a string, and strings are truthy. For example, if: true && ${{ false }} will evaluate to true.

The part of the docs on [expressions](https://docs.github.com/en/actions/learn-github-actions/expressions#operators) also has the below buried in it (can't link directly so scroll down to the table)

String | Parsed from any legal JSON number format, otherwise NaN.Note: empty string returns 0.
-- | --

Reply:
you can also manually launch a run of sanity tests by going to the sanity tests workflow in the Actions page and choosing the branch and user to run with. That may help with debugging

Reply:
changed to a truthiness check.